sparse_strips/vello_sparse_shaders/shaders/render_strips.wgsl

```diff
@@ -56,6 +56,25 @@ const COMPOSE_XOR: u32 = 11u;
 const COMPOSE_PLUS: u32 = 12u;
 const COMPOSE_PLUS_LIGHTER: u32 = 13u;
 
+// Mix modes
+const MIX_NORMAL = 0u;
+const MIX_MULTIPLY = 1u;
+const MIX_SCREEN = 2u;
+const MIX_OVERLAY = 3u;
+const MIX_DARKEN = 4u;
+const MIX_LIGHTEN = 5u;
+const MIX_COLOR_DODGE = 6u;
+const MIX_COLOR_BURN = 7u;
+const MIX_HARD_LIGHT = 8u;
+const MIX_SOFT_LIGHT = 9u;
+const MIX_DIFFERENCE = 10u;
+const MIX_EXCLUSION = 11u;
+const MIX_HUE = 12u;
+const MIX_SATURATION = 13u;
+const MIX_COLOR = 14u;
+const MIX_LUMINOSITY = 15u;
+const MIX_CLIP = 128u;
```

Comment:
Is `MIX_CLIP` ever used? If it isn't, we could reduce the number of bits reserved for mixes from 7 to 4 IIUC

Reply:
Edit: If it isn't used, is it also worth keeping around?

Reply:
Are you able to add a TODO if it's a far future exercise so we don't forget?